Source/WebCore/platform/graphics/gstreamer/InbandTextTrackPrivateGStreamer.cpp

```diff
@@ -103,18 +103,16 @@ void InbandTextTrackPrivateGStreamer::notifyTrackOfSample()
             GST_WARNING("Track %d got sample with no buffer.", m_index);
             continue;
         }
-        GstMapInfo info;
-        gboolean ret = gst_buffer_map(buffer, &info, GST_MAP_READ);
-        ASSERT(ret);
-        if (!ret) {
+        GstMappedBuffer bufferMapped(buffer, GST_MAP_READ);
```

Comment:
This is just a big IF coming from a non-native English speaker to a native English speaker. Shouldn't all these cases have the "mapped" in advance, like mappedBuffer, mappedInitData, mappedKeyIDBuffer, etc?